ROADMAP.md

```diff
@@ -0,0 +1,74 @@
+# Rust Analyzer Roadmap 01
+
+Written on 2018-11-06, extends approximately to February 2019.
+After, we should coordinate with rustc/RLS developers to align goals and share code and experience.
```

Comment:
Reading between the lines your actual goal is "impress people at Rust All Hands", right? ;)

Reply:
Not really: IntelliJ Rust, at this moment, is definitely more impressive then rust analyzer. 

It's more than I have a feeling that RLS moves slowly in the wrong direction, and I'd love to somehow overcome that inertia :)

Reply:
And that's very good!

(I meant not to pitch it in terms of features that work but in the sense of: Here's this project, and because we have reached these goals we are confident in the architecural decisions and think it's worthwhile to persue this further/make this a priority for 2019.)